### PR TITLE
feat(telepony.line.calls): add tooltip on simultaneous lines

### PR DESCRIPTION
**Pull request by Steffy29**
Ref: UXCT-88

### CODE REVIEW

File: client/app/telecom/telephony/line/calls/simultaneousLines/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -60,4 +60,5 @@
    <translation id="telephony_line_actions_line_calls_trunk_simultaneous_channels_price_per_month" qtlid="479837">/ par mois</translation>
    <translation id="telephony_line_actions_line_calls_trunk_simultaneous_channels_bandwidth_rate" qtlid="479850">{{bandwidth}} Kbps</translation>
 
+   <translation id="telephony_line_actions_line_calls_simultaneous_tooltip">Pour le nombre d'appels simultanés maximum par téléphone, veuillez vous référer à la documentation de votre téléphone.</translation>
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/line/calls/simultaneousLines/translations/Messages_fr_FR.xml]:
```suggestion
   <translation id="telephony_line_actions_line_calls_simultaneous_tooltip">Pour le nombre d'appels simultanés maximum par téléphone, veuillez-vous référer à la documentation de votre téléphone.</translation>
```
Also something like `Le nombre d'appels simultanés est dépendant de votre matériel, veuillez-vous référer à sa documentation plus d'informations.` would be better, don't you think ?

File: client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html [hunk 2]
```diff
@@ -36,14 +36,19 @@ <h1 data-translate="telephony_line_actions_line_calls_simultaneous_line_title"><
                     <div class="row my-4">
                         <div class="col-xs-12 col-md-12">
                             <label class="control-label"
-                                   for="simultaneousLines">
+                                   for="simultaneousLines"
+                                   data-tooltip-placement="top-end"
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html]: Please use https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/tooltip instead

File: client/app/telecom/telephony/line/calls/simultaneousLines/translations/Messages_fr_FR.xml (hunk 1 above)
Comment (Steffy29) [reply client/app/telecom/telephony/line/calls/simultaneousLines/translations/Messages_fr_FR.xml]: yes, it's better.

File: client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html [hunk 3]
```diff
@@ -36,14 +36,19 @@ <h1 data-translate="telephony_line_actions_line_calls_simultaneous_line_title"><
                     <div class="row my-4">
                         <div class="col-xs-12 col-md-12">
                             <label class="control-label"
-                                   for="simultaneousLines">
+                                   for="simultaneousLines"
+                                   data-oui-tooltip-placement="top-end"
+                                   data-oui-tooltip-append-to-body="true"
+                                   data-uib-tooltip="{{ ::'telephony_line_actions_line_calls_simultaneous_tooltip' | translate }}">
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html]: You are not using the `oui tooltip` there, Bootstrap is still handling it :)

File: client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html [hunk 4]
```diff
@@ -36,14 +36,19 @@ <h1 data-translate="telephony_line_actions_line_calls_simultaneous_line_title"><
...
                                 <span data-translate="telephony_line_actions_line_calls_simultaneous_line_label"></span>
                                 <span data-ng-if="LineSimultaneousLinesCtrl.options.details"
                                       data-translate="telephony_line_actions_line_calls_simultaneous_details"
                                       data-translate-values="{
                                           'basic': LineSimultaneousLinesCtrl.options.details.basic,
                                           'extra': LineSimultaneousLinesCtrl.options.details.extra
                                       }"></span>
+                                <i class="oui-icon oui-icon-help_circle ml-1"
```
Comment (FredericEspiau) [inline client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html]: Could you please realign and change the `i` into a `span` ?

File: client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html [hunk 5]
```diff
@@ -36,14 +36,18 @@ <h1 data-translate="telephony_line_actions_line_calls_simultaneous_line_title"><
                     <div class="row my-4">
                         <div class="col-xs-12 col-md-12">
                             <label class="control-label"
-                                   for="simultaneousLines">
+                                   for="simultaneousLines"
+                                   oui-tooltip-placement="top-end"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html]: Missing some 'data-' prefixes.

Comment (Steffy29) [reply client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html]: done

File: client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html [hunk 6]
```diff
@@ -36,14 +36,18 @@ <h1 data-translate="telephony_line_actions_line_calls_simultaneous_line_title"><
                     <div class="row my-4">
                         <div class="col-xs-12 col-md-12">
                             <label class="control-label"
-                                   for="simultaneousLines">
+                                   for="simultaneousLines"
+                                   data-oui-tooltip-placement="top-end"
+                                   data-oui-tooltip-append-to-body="true"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html]: According to the [`oui-tooltip`](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/tooltip#api) documentation, there is no `append-to-body` attribut available. Could you please remove it?

**Pull request merged**